### PR TITLE
(feat) add CKEditor component

### PR DESCRIPTION
**Pull request by daniel-v**
Please CR the editor

### CODE REVIEW

File: test/ckeditor_test.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Title</title>
+        <script defer src="https://cdnjs.cloudflare.com/ajax/libs/ckeditor/4.7.0/ckeditor.js"></script>
+        <link rel="x-dart-test" href="app_test.dart">
```
Comment (valakis) [inline test/ckeditor_test.html]: href should be `ckeditor_test.dart`

File: lib/src/components/ckeditor/ckeditor.dart [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+/// __Properties:__
+/// - `editor-name: String` -- element CSS selector to replace with CKEditor
+/// - `extra-plugins: List<ExtraPlugin>` -- extra plugins to load with CKEditor
+/// - `config-url: String` -- url of the config file to load for CKEditor
+@Component(selector: 'skawa-ckeditor', templateUrl: 'ckeditor.html', changeDetection: ChangeDetectionStrategy.OnPush)
+class SkawaCkeditorComponent implements AfterViewInit, OnDestroy {
+  final _changeController = new StreamController<String>.broadcast();
+  _CKEditor _ckeditor;
+
+  @Input('editor-name')
```
Comment (valakis) [inline lib/src/components/ckeditor/ckeditor.dart]: mindenhol az input is lower camel case volt eddig

File: test/ckeditor_test.html (hunk 1 above)
Comment (valakis) [inline test/ckeditor_test.html]: href should be ckeditor_test.dart

**Comment by daniel-v** [comment]
Requested changes should be fixed now. Let's see what Travis says.

**Pull request merged**